snapcraft.yaml

```diff
@@ -140,6 +144,7 @@ parts:
       bin/subiquity-service: usr/bin/subiquity-service
       bin/subiquity-server: usr/bin/subiquity-server
       bin/subiquity-cmd: usr/bin/subiquity-cmd
+      bin/subiquity-umockdev-wrapper: system_scripts/subiquity-umockdev-wrapper
```

Comment:
Unfortunately usr/bin/ has the snap vendored python3 version we're trying to avoid in the first place. We could get around this on a per script basis, but my thought was to have the single system_scripts_env for all of these types of scripts.

Reply:
got it, skip this suggestion then